internal/envoy/v3/auth.go

```diff
@@ -91,6 +91,15 @@ func validationContext(ca []byte, subjectName string, skipVerifyPeerCert bool) *
 		}
 	}
 
+	if len(crl) > 0 {
+		vc.ValidationContext.Crl = &envoy_api_v3_core.DataSource{
+			Specifier: &envoy_api_v3_core.DataSource_InlineBytes{
+				InlineBytes: crl,
```

Comment:
nit: could you add a TODO comment to move this to SDS (similar to https://github.com/projectcontour/contour/blob/f44a2125a71c240fca1fa03aee2fa57694c5a28e/internal/envoy/v3/auth.go#L74) which we should do eventually

Reply:
I moved the comment to function level, since it seems the whole `ValidationContext` seems to be delivered via SDS ([doc](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret#sds-configuration)) and not just the DataSource(s): CA and CRL.